synfig-studio/src/synfigapp/actions/valuedescset.cpp

```diff
@@ -98,7 +98,8 @@ Action::ValueDescSet::ValueDescSet():
 	time(0),
 	recursive(true),
 	animate(false),
-	lock_animation(false)
+	lock_animation(false),
+	add_waypoint_initial(false)
```

Comment:
couldn't the `animate` parameter be used?

Reply:
Looks like it definitely can, done.